source/ui/CLS/CLSButtonBar.cpp

```diff
@@ -81,23 +81,31 @@ void CLSButtonBar::removeButton(QAbstractButton *button)
 	}
 }
 
+void CLSButtonBar::removeButtons()
```

Comment:
It depends on how the buttons are used. Because another widget could add buttons to the button bar and still keep references to those buttons, there would be a problem when those buttons are deleted. In that case, the widget would want to remove buttons instead of clearing them. It doesn't hurt to provide both functionalities. That's what I was thinking, anyways.